apps/condo/domains/resident/schema/DiscoverServiceConsumersService.js

```diff
@@ -1,17 +1,26 @@
 /**
  * Generated by `createservice resident.DiscoverServiceConsumersService --type mutations`
  */
+const { get, flatMap } = require('lodash')
 
-const { get, _ } = require('lodash')
-
+const { featureToggleManager } = require('@open-condo/featureflags/featureToggleManager')
 const { getLogger } = require('@open-condo/keystone/logging')
 const { GQLCustomSchema } = require('@open-condo/keystone/schema')
 
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
 const { BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+const { ENABLE_DISCOVER_SERVICE_CONSUMERS } = require('@condo/domains/common/constants/featureflags')
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const { SERVICE_PROVIDER_TYPE } = require('@condo/domains/organization/constants/common')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
 const access = require('@condo/domains/resident/access/DiscoverServiceConsumersService')
 const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+
 const logger = getLogger('DiscoverServiceConsumersMutation')
 
+const asyncFilter = async (arr, predicate) => Promise.all(arr.map(predicate)).then((results) => arr.filter((_v, index) => results[index]))
```

Comment:
What's `_v` for?

Reply:
This function is not used anymore. Deleted.